apps/api/app/api/v1/endpoints/notification.py

```
@@ -301,3 +305,63 @@ async def unregister_device_token(
     except Exception as e:
         logger.error(f"Failed to unregister device token: {e}")
         raise HTTPException(status_code=500, detail=str(e))
+
+
+@router.get("/notifications/preferences/channels", response_model=ChannelPreferences)
+async def get_channel_preferences(
+    current_user: dict = Depends(get_current_user),
```

Ruff (0.15.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)